examples/samplers/fast_sampling_with_jax_and_numba.ipynb

```diff
@@ -19,18 +19,58 @@
    "cell_type": "markdown",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #11.        multiprocessing.set_start_method("spawn", force=True)
</pre><p>Comment on why this is necessary?</p><p> </p><p>Also would be good to add a warning filter for the <code>RuntimeWarning: os.fork() was called. os.fork() </code>  that jax spams out</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/794/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p>Ditching this.</p>